main.go

```diff
@@ -53,6 +53,8 @@ Options:
   --kubelet-port <port>                      Override kubelet port for
                                               automatically discovered nodes.
                                               [default: 10255]
+  --kubelet-insecure                         Insecure skip SSL verify.
+  --kubelet-root-ca-cert <filepath>          Filepath to root CA cert.
```

Comment:
The default ca filepath can be used here (e.g. /var/run/secrets/kubernetes.io/serviceaccount/ca.crt).
https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/

Reply:
This default ca is mounted only for pods using the default service account, we don't. Is this still valid for us?

> The CA certificate bundle is automatically mounted into pods using the default service account at the path /var/run/secrets/kubernetes.io/serviceaccount/ca.crt. If you are not using the default service account, ask a cluster administrator to build a configmap containing the certificate bundle that you have access to use.

Reply:
@YasserRabee I can still see the file inside our pod. @AmrEleraky any thoughts?

Reply:
@mie00 maybe I'm understanding the docs incorrectly. I see the file there, so I'm gonna use. Thanks